src/components/Text/index.tsx

```diff
@@ -32,6 +34,7 @@ const Text: React.FC<Props> = ({
           ? theme.lineHeights[0]
           : theme.lineHeights[1],
       marginBottom: leading ? theme.spacing[leading] : theme.spacing[size - 1],
+      opacity: label ? 0.5 : 1,
```

Comment:
Could you break this change into a separate commit?

Reply:
The purpose of the label prop is to apply these styles. Maybe changing the commit message would help, or I could also break it out into a separate commit.